diff --git a/node_modules/react-native-worklets/lib/module/memory/serializable.native.js b/node_modules/react-native-worklets/lib/module/memory/serializable.native.js
index d82f096..9270421 100644
--- a/node_modules/react-native-worklets/lib/module/memory/serializable.native.js
+++ b/node_modules/react-native-worklets/lib/module/memory/serializable.native.js
@@ -275,7 +275,7 @@ function cloneArray(value, shouldPersistRemote, depth) {
   return clone;
 }
 function cloneNonWorkletFunction(fun) {
-  const clone = WorkletsModule.createSerializableNonWorkletFunction(fun, __DEV__ ? fun.name : undefined);
+  const clone = WorkletsModule.createSerializableNonWorkletFunction(fun, undefined);
   if (globalThis.WeakRef) {
     // WeakRef is installed on runtimes only with Hermes microtaskQueue enabled.
     serializableMappingCache.set(fun, new WeakRef(clone));
diff --git a/node_modules/react-native-worklets/src/memory/serializable.native.ts b/node_modules/react-native-worklets/src/memory/serializable.native.ts
index 5417b6e..abce467 100644
--- a/node_modules/react-native-worklets/src/memory/serializable.native.ts
+++ b/node_modules/react-native-worklets/src/memory/serializable.native.ts
@@ -405,7 +405,7 @@ function cloneNonWorkletFunction<TArgs extends unknown[], TReturn>(
 ): SerializableRef<(...args: TArgs) => TReturn> {
   const clone = WorkletsModule.createSerializableNonWorkletFunction(
     fun,
-    __DEV__ ? fun.name : undefined
+    undefined
   ) as SerializableRef<(...args: TArgs) => TReturn>;
 
   if (globalThis.WeakRef) {
